app/models/relationship.rb

```diff
@@ -0,0 +1,7 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
+  belongs_to :followed, class_name: "User"
```

Comment:
```suggestion
  belongs_to :follower, class_name: User.name
  belongs_to :followed, class_name: User.name
```
E fix tương tự cho các class_name khác nha